tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1454,6 +1454,32 @@ Here are some examples of expressions::
 Identifiers, attributes, and subscripts are valid assignment targets. Other expressions cannot be on the left side of
 an assignment.
 
+self
+^^^^
+
+``self`` can be used to refer to the current instance and is often equivalent to directly referring to symbols available in
+the current script, however, while this is generally considered a bad practice, ``self`` also allows you to access properties,
+methods, and other names that are defined dynamically (i.e. are expected to exist in subtypes of the current script, or are
```

Comment:
Accessing dynamically defined properties is not a bad practice, it's a different approach, a different paradigm. But accessing members of child classes in the base class can be considered bad practice.

Also "in the current script" can be confusing, since there are inner classes, I would prefer "in the current class".

Reply:
Done in 22b1ea7db!